pkg/forth/forth.go

```diff
@@ -64,50 +80,51 @@ var opmap = map[string]ForthOp{
 // meaning Length is 0), Newop (insert a new or replacement operator),
 // and Reset (clear the stack, mainly diagnostic)
 type Forth interface {
-	Push(string)
-	Pop() string
+	Push(interface{})
```

Comment:
should the type for the arg to push be a Cell instead of an interface{}.

feel free to disregard, i'm barely Go-literate.

Reply:
That's a good question. I decided in the end to make it interface{} since it's an external caller doing this. But I think for more correctness it should be Cell. I'll change it.